services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -141,6 +143,8 @@ protected void doControllerMsg(CommandMessage message) {
 
     private void handleCommand(CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
+        logger.debug("Handling message: '{}'. Topic: '{}'. Destination: '{}'.",
```

Comment:
message is unclear - topic and destination is a "reply topic" and "reply destination".